kanidmd_web_ui/src/components/change_unix_password.rs

```diff
@@ -170,8 +177,10 @@ impl Component for ChangeUnixPassword {
                       onsubmit={
                         ctx.link().callback(|e: FocusEvent| {
                           e.prevent_default();
-                          let form = e.target().and_then(|t| t.dyn_into::<HtmlFormElement>().ok()).unwrap();
-                          Msg::Submit(FormData::new_with_form(&form).unwrap())
+                          #[allow(clippy::expect_used)]
+                          let form = e.target().and_then(|t| t.dyn_into::<HtmlFormElement>().ok()).expect("Failed to pull the form data from the browser");
```

Comment:
We can probably swap to expect_throw here instead?

Reply:
Fixed in bfc92d1b